pwiz_tools/Skyline/FileUI/PeptideSearch/PeptideSearchResources.resx

```diff
@@ -365,4 +365,7 @@ Click 'Retry' to try building again after placing the original spectrum files in
   <data name="SearchControl_CanWizardClose_Cannot_close_wizard_while_the_search_is_running_" xml:space="preserve">
       <value>Cannot close wizard while the search is running. Do you want to cancel the search?</value>
   </data>
+  <data name="ImportPeptideSearchDlg_AddDetectedFeaturesToDocument_Did_not_detect_any_features___check_your_settings_and_try_again" xml:space="preserve">
+      <value>Did not detect any features - check your settings and try again</value>
```

Comment:
I would prefer "No features were detected. Check your settings and try again." Possibly with a new line between the sentences. This was our standard format for a while in Visual C++. Line 1: state the problem, Line 2: Suggest a possible solution. It is not standard in Skyline, though. But at least the goal is similar and having two separate sentences seems better to me.

Reply:
Sounds good.